connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -199,12 +217,12 @@ public void checkCORSRequest(String corsDomain, String origin, String expectedHe
         HttpRequest request = new HttpGet("/connectors");
         request.addHeader("Referer", origin + "/page");
         request.addHeader("Origin", origin);
-        CloseableHttpClient httpClient = HttpClients.createMinimal();
+        httpClient = HttpClients.createMinimal();
         HttpHost httpHost = new HttpHost(
             server.advertisedUrl().getHost(),
             server.advertisedUrl().getPort()
         );
-        CloseableHttpResponse response = httpClient.execute(httpHost, request);
+        response = httpClient.execute(httpHost, request);
```

Comment:
We still have to close this `response` since it gets overwritten by a different one before the test completes.

Reply:
Changed for local try-with-resources.